dist/node-connector.js

```diff
@@ -17162,6 +17348,48 @@ module.exports =
 	      this._client[0].detect_column_types(this._sessionId[0], fileName, thriftCopyParams, callback);
 	    }
 
+	    /**
```

Comment:
Is there a reason to have the jsdocs not directly above the actual method, for example here just above line 16795?

Reply:
This is a compiled file. I'm not sure why it laid the comments out this way, or why this compiler isn't removing them and minifying.